Python/datacube.py

```diff
@@ -13,56 +12,19 @@ def read_data(data_path, dv_key_name):
         dv_key_name (Str): name of the design value as 
             in the netCDF file
     Returns:
-        ds (xarray Dataset): data cube of assembled ensemble models 
+        xr (xarray Dataset): data cube of assembled ensemble models 
```

Comment:
You have a module that is imported and named `xr` and a variable named `xr`.  I'm not sure what the behaviour in python is for this, but you may want to change the name to avoid the problem.